src/kernel/mconfig.ml

```diff
@@ -358,6 +360,14 @@ let merlin_flags = [
     marg_path (fun path merlin -> {merlin with stdlib = Some path}),
     "<path> Change path of ocaml standard library"
   );
+  (
+    "-cache-time",
+    Marg.param "int" (fun prot merlin ->
+        try {merlin with cache_time = (int_of_string prot)}
+        with _ -> invalid_arg "Valid value is int";
+      ),
+    "Change file cache retention time. Measured in minutes."
```

Comment:
Might be useful to document the default value (5) here.

Reply:
Done